t/t.go

```diff
@@ -109,7 +109,7 @@ func command(args ...string) *exec.Cmd {
 
 func startCluster(composeFile, prefix string) error {
 	cmd := command(
-		"docker-compose", "-f", composeFile, "-p", prefix,
+		"docker-compose", "--compatibility", "-f", composeFile, "-p", prefix,
```

Comment:
I think we should add this everywhere in `t.go`, even inside `stopCluster` (I'm guessing this may affect the `./t -r` cleanup process.

Reply:
added in latest commit